ocaml/typing/persistent_env.ml

```diff
@@ -392,6 +411,9 @@ let is_imported {imported_units; _} s =
 let is_imported_opaque {imported_opaque_units; _} s =
   CU.Name.Set.mem s !imported_opaque_units
 
+let is_parameter_unit penv s =
+  is_registered_parameter_import penv s
```

Comment:
Why are there two names for the same function? Are they different in later patches?

Reply:
Looks like I lost track of what the function was supposed to do. The doc comment in the .mli is just wrong now. I should probably just delete this function and add `is_registered_parameter_import` to the .mli.

Reply:
Done.